### ISSUE TITLE
[UWP][Accessibility] Name attribute is not defined for 'Spider' image under 'Solitaire.JSON' list item.

### ISSUE THREAD

**Issue by almedina-ms** [microsoft/AdaptiveCards#3927]
# Platform
* UWP

# Version of SDK
master

# Details

Repro Steps:​
1. Launch the application.​
2. New screen starts appearing. Navigate to Left pane JSOn files.​
3. Now select "Solitaire.JSON" button. It will update the Middle and Right Pane. Navigate all the elements of both the panes.​
4. Verify alt of 'Spider' image control by using accessibility insights for desktop.

Actual Result:
Name attribute is not defined for 'Spider' image under 'Solitaire.JSON' list item and screen reader is reading as image while navigating in scan mode, due to this user will not get the purpose of image control which presents in right pane.

Expected Result:
Name attribute should be define for 'Spider' image under 'Solitaire.JSON' list item and screen reader should be read as 'Spider' image while navigating in scan mode, so that user can easily understand the purpose of image control which presents in right pane.
